Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN chmod +x /wait /entry
 
 WORKDIR /usr/app
 COPY . .
-RUN npm ci
+RUN npm install
```

Comment:
why is that changed back ?

Reply:
It is changed back because I'd like to buidl the container in the `node_modules/@daostack/subgraph` repo, but the intallation of npm does not include the `package-lock.json`. And so `npm ci` fails. Unfortunately, because it would be better/more predictable.

Reply:
@jellegerbrandy We should instead publish the image to dockerhub and use it in the `docker-compose.yml` to both avoid this problem and shorten build times.

Reply:
Yes I agree.

Reply:
Actually, you are just using this container to run the tests in, right? (Is that a good idea?) In that case, there would be no need to build that container at all...